arangod/Agency/AgencyCommon.h

```diff
@@ -40,6 +40,8 @@ static std::string const NO_LEADER = "";
 
 enum role_t { FOLLOWER, CANDIDATE, LEADER };
 
+enum apply_ret_t {APPLIED, PRECONDITION_FAILED, FORBIDDEN, UNKNOWN_ERROR};
```

Comment:
Was erlaube?

Reply:
This is a means to allow application of requests to the state machine to respond on a more fine grained level that just true/false